zerver/lib/upload.py

```diff
@@ -445,15 +447,15 @@ def upload_avatar_image(user_file, user_profile, email):
     # type: (File, UserProfile, Text) -> None
     upload_backend.upload_avatar_image(user_file, user_profile, email)
 
+def upload_message_image(uploaded_file_name, uploaded_file_size, content_type, file_data, user_profile, target_realm=None):
+    # type: (Text, int, Optional[Text], binary_type, UserProfile, Optional[Realm]) -> Text
+    return upload_backend.upload_message_image(uploaded_file_name, uploaded_file_size, content_type, file_data,
+                                               user_profile, target_realm=target_realm)
+
 def upload_icon_image(user_file, user_profile):
     # type: (File, UserProfile) -> None
     upload_backend.upload_realm_icon_image(user_file, user_profile)
 
-def upload_message_image(uploaded_file_name, content_type, file_data, user_profile, target_realm=None):
-    # type: (Text, Optional[Text], binary_type, UserProfile, Optional[Realm]) -> Text
-    return upload_backend.upload_message_image(uploaded_file_name, content_type, file_data,
-                                               user_profile, target_realm=target_realm)
-
```

Comment:
Can you undo the moving of this function?  

Also might make sense to wrap the function arguments to 2 lines.